<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="find_start">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕 Semilight</family>
    </font>
   </property>
   <property name="text">
    <string>check!</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sta_code">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕 Semilight</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="Sta_name">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕 Semilight</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="Line_nam">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕 Semilight</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕 Semilight</family>
    </font>
   </property>
   <property name="text">
    <string>노선명</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕 Semilight</family>
    </font>
   </property>
   <property name="text">
    <string>역 번호</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="sta_eta_time">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕 Semilight</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕 Semilight</family>
    </font>
   </property>
   <property name="text">
    <string>인접시간</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_sta">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕 Semilight</family>
    </font>
   </property>
   <property name="text">
    <string>인접 시간표</string>
   </property>
  </widget>
  <widget class="QSlider" name="day_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>20</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕 Semilight</family>
    </font>
   </property>
   <property name="maximum">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕 Semilight</family>
    </font>
   </property>
   <property name="text">
    <string>평일</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕 Semilight</family>
    </font>
   </property>
   <property name="text">
    <string>토요일</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕 Semilight</family>
    </font>
   </property>
   <property name="text">
    <string>공휴일</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕 Semilight</family>
    </font>
   </property>
   <property name="text">
    <string>역명</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="sta_timetable2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>621</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
